### PR TITLE
Add Jani lobby background

### PR DESCRIPTION
**Pull request by Psychpsyo**
## About the PR

I've added a new lobby background.
Drew it myself. :D

## Why / Balance

Because janitors are currently woefully underrepresented in the lobby screens, compared to sec and clowns, lol

## Technical details

None

## Media

image

ss14lobby

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

no

**Changelog**

:cl:
- add: New lobby background art!

### CODE REVIEW

File: Resources/Textures/LobbyScreens/attributions.yml [hunk 1]
```diff
@@ -46,4 +46,9 @@
 - files: ["justaweekaway.webp"]
   license: "CC-BY-SA-3.0"
   copyright: "plantyfern on discord"
+  source: "https://github.com/space-wizards/space-station-14"
+
+- files: ["janishootout.webp"]
+  license: "CC0"
```
Comment (Tayrtahn) [inline Resources/Textures/LobbyScreens/attributions.yml:52]:
```suggestion
  license: "CC0-1.0"

```

Comment (Psychpsyo) [reply Resources/Textures/LobbyScreens/attributions.yml]: Just noticed myself and fixed it now.

**Review by Tayrtahn — APPROVED** [review]

**Comment by beck-thompson** [comment]
Looks sick! Amazing job 👍

**Comment by lzk228** [comment]
literally me

**Comment by EmoGarbage404** [comment]
very epic

**Pull request merged**